spark-dockerfile/README.md

```diff
@@ -0,0 +1,19 @@
+## Usage :
+```
+docker run -ti \
+       -p "4040:4040" \
+       gisaia/spark:latest \
+       spark-shell \
+       ...
+```
+
+## Publish the docker image in DockerHub :
+```
+docker build -t gisaia/spark:x.x.x -t gisaia/spark:latest .
+```
+```
+docker login
+```
+```
+docker push gisaia/spark
```

Comment:
I think when no tag is given on a `docker push`, it will default to pushing tag `latest`, and only this `latest`. To also push the versioned image, I think it should be:

```bash
docker push gisaia/spark:latest
docker push gisaia/spark:x.x.x
```